docs/src/user/manual/help.md

```diff
@@ -0,0 +1,82 @@
+---
+title: BIMS Documentation
+summary: Central documentation site for BIMS projects
+    - Tim Sutton
+    - Jeremy Prior
+    - Ketan Bamniya
+date: 26-09-2023
+some_url: https://github.com/kartoza/bims-website
+copyright: Copyright 2023, Kartoza
+contact: 
+license: This program is free software; you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option) any later version.
+#context_id: 1234
+---
+
+# Help
+
+[![Home Page](./img/help-img-3.png)](./img/help-img-3.png)
+
+
+## Getting help using the FBIS platform
+
+We are so pleased that users are using FBIS! The site offers a broad range of features, which are documented in the user [guide](../guide/index.md) and [manual](./index.md). For a quick start, users can explore the walk-through video of the platform. Lastly, although we strive to provide a seamless experience, users may occasionally encounter issues with the site. If such an issue arises, users are encouraged to report it on our issue tracker, and we will do our best to resolve it promptly!
+
+## FBIS Testing site
+
+For practicing the workflows for capturing data on FBIS, users should use the [FBIS Testing site](https://testing.healthyrivers.kartoza.com/). Here, users can add dummy data, as the data are overwritten with a mirror image of the FBIS Live (production) site every week. The same login works for both `FBIS Live` and `FBIS Testing`.
+
+## User Manual
+
+User can also checkout the [user manual](https://freshwaterbiodiversity.org/uploaded/documents/document/MmWwDIn-SCmrqt_lyQVbVw.pdf) for more information about the platform.
+
+## instructional Videos
+
+We have also created a series of instructional videos to help users get started with the platform.
+
+* **Introduction to key FBIS components:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=aex5tzLNWAo)
```

Comment:
We are able to embed videos using the `mkdocs-video` plugin using `![type:video](video_url.com)`.